doc/man1/openssl-s_server.pod.in

```diff
@@ -792,6 +793,15 @@ instead of BIO_write() to send the HTTP response requested by a client.
 This option is only valid when B<-ktls> along with B<-WWW> or B<-HTTP>
 are specified.
 
+=item B<-zerocopy_sendfile>
+
+If this option is set, SSL_sendfile() will use the zerocopy TX mode, which gives
+a performance boost when used with kTLS hardware offload. Note that invalid
```

Comment:
I am not seeing "kTLS" (note capitalization) anywhere else in the documentation, except this PR.

Please change here, and elsewhere, to "Kernel TLS" or "KTLS" (all caps).